packages/search/src/features/reindex/handler.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
+ */
+import * as Hapi from '@hapi/hapi'
+import { prune } from './prune'
+import { reindex, updateAliases } from './reindex'
+
+export async function reindexHandler(
+  request: Hapi.Request,
+  h: Hapi.ResponseToolkit
+) {
+  const { timestamp } = request.payload as { timestamp: string }
+
+  const { index } = await reindex(timestamp)
+  await updateAliases()
+
+  prune()
+
+  return h
+    .response({
+      status: 'completed',
+      message: `ElasticSearch reindexing completed for ${index}`,
+      index
+    })
+    .code(200)
```

Comment:
We need to test this with having a lot of records. I'm not sure how long the streaming actually takes, so should this be added into Redis with a random uuid, and then having a status-endpoint which tells if it is running or failed or completed.

Reply:
We can roll with this for now. Fetching from the migrations shouldn't timeout as fetch doesn't support timeout, and I tested that this takes around 3.5 seconds per 1000 records.